.github/workflows/release.yml

```diff
@@ -90,3 +107,30 @@ jobs:
         run: yarn build
       - name: Test
         run: yarn test
+
+  publish-npm:
+    name: 'Publish package to NPMJS'
+    needs:
+      - preconditions
+      - lint
+      - dependency-check
+      - test
+      - check-version
+    runs-on: ubuntu-latest
+    if: ${{ needs.check-version.outputs.is_new_version == 'true' }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@digicatapult'
+      - name: Change Folder
+        run: cd packages/components
+      - name: Install packages
+        run: npm ci
+      - name: Publish to npmjs packages
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
these need to be done in a single script not three. Each time the `cwd` will be reset